packages/components/Molecules/Card/CardEmergencyContact.vue

```diff
@@ -12,6 +12,7 @@
         :href="filterLink"
         :innerWrap="false"
         v-if="link"
+        :ariaLabelledBy="id"
```

Comment:
See comment on link component: you can just add `aria-labelledby` here without having to pass it as a prop and it will be added to the root element of the link component (the a element in this case)